doc/docs/modules/ROOT/pages/aura.adoc

```diff
@@ -25,3 +25,11 @@ driver URI is provided by AuraDB itself when you create a database.
 AuraDB customers connecting from Databricks may encounter SSL handshake errors due to Databricks' custom Java security settings removing certain cipher support.
 
 See the AuraDB support article link:{url-aura-kbase-databricks}[Connecting to Aura with Databricks] for more information and instructions on how to configure your Databricks cluster to support connections to AuraDB.
+
+== Combining AuraDB and AuraDS
+
+AuraDB is the Neo4j Cloud solution for OLTP processes while AuraDS is designed for managing
```

Comment:
A general comment about docs style. I'm not sure how core db does it, but in GDS we use "one sentence per line" in docs, which is way easier to review, maintain and also write.